tools/CoqMakefile.in

```diff
@@ -39,6 +39,7 @@ CAMLP4O           := $(COQMF_CAMLP4O)
 CAMLP4BIN         := $(COQMF_CAMLP4BIN)
 CAMLP4LIB         := $(COQMF_CAMLP4LIB)
 CAMLP4OPTIONS     := $(COQMF_CAMLP4OPTIONS)
+CAMLFLAGS         := $(COQMF_CAMLFLAGS)
```

Comment:
What happens if the user overrides this variable?

Reply:
You are welcome to be more explicit here:
  https://github.com/coq/coq/blob/master/doc/refman/RefMan-uti.tex#L132
and suggest to use `CAMLFLAGS+=extra`

Reply:
So not possible to do `make CAMLFLAGS=foo` ?

Reply:
Hum, I think that "we" know better the flags to pass to `ocamlc`, e.g. we know that we must pass `-safe-string` or things will break (that is the point of this PR). IMHO coq_makefile should work for the lambda user, and I quite don't see why they would like to override completely the flags.  I mean, adding a -I or something like that is still possible (and somewhat documented).

More in general, I did not document nor think too much about which flags can be overridden on the command line. The only few that are documented/thought-about are the timing ones and VERBOSE.  I guess we should say explicitly that all variables not documented as "overrideable" are not supposed to be overridden (in the doc/makefile header).